src/evaluators/response/PHAL_ResponseSquaredL2DifferenceSide_Def.hpp

```diff
@@ -70,6 +100,9 @@ ResponseSquaredL2DifferenceSideBase(Teuchos::ParameterList& p, const Teuchos::RC
 
   this->setName("Response Squared L2 Error Side" + PHX::print<EvalT>());
 
+  if(plist->isParameter("Response Depends On Solution Column") && plist->get<bool>("Response Depends On Solution Column"))
+    cell_topo = p.get<Teuchos::RCP<Teuchos::ParameterList> >("Parameters From Problem")->get<Teuchos::RCP<const CellTopologyData> >("Cell Topology");
```

Comment:
Is the template arg wrong? Shouldn't it be `Teuchos::ParameterList*` (notice the pointer missing in your version)?

Reply:
I think it's correct, it's already and RCP. Why are you expecting it to be an RCP of a pointer?

Reply:
Ah, I missed the RCP part. Sorry.